_getting-started/data-sample-overview.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Data Sample Overview
+layout: article
+section: Basic Concepts
+order: 1
+---
+
+This document provides basic information on [data samples](#data-samples) and how they are useful when [building integration flows](#samples-in-integraton-flows). Additionally, it gives high-level information about [exceptions](#exceptions).
+
+## Data Sample
+
+
+A data sample is an example of [component](/integration-component) output data.
+It is essential in building integration flows, because it allows you to see what
+sort of data the next component in the flow will receive from the previous one.
+With this information you can configure the next component to act properly on
+the input it receives.
+
+You can generate more than one sample from a component. By default, the first one in the list will be used. Also, you can manually select the proper sample from the list. All unused samples will be deleted once you make a choice.
+
+## Samples in Integration Flows
+
+
+Each component in a flow has its own input/output data standards. For proper
+flow operation, the components need [data mapping](//guides/mapping-data). It is a process of
+data conversion from one component’s standard to the next one’s. Basically, data
+mapper retrieves data from one component and relays it to the next component in
+a compatible form. In order to see what sort of output data the first component
+produces we use data sample.
+
+A data sample can be retrieved from a component or written manually by the
+integrator. In any case, the sample is then given as input to the next
+component. The receiving component may not require all the data from the sample
+or may require it in completely different order. That’s when you configure
+proper mapping so that the receiving component gets input in accordance with its
+standard.
+
+**NOTE:** Samples exist for flow building and testing purposes. However, the data from samples will go into production when you generate a sample. If you want to avoid this, you can use **Generate Stub Sample** function. It retrieves the component's output template and fills it with fake data that you can discard afterwards.
```

Comment:
> However, the data from samples will go into production when you generate a sample.

I think I know what you are saying here but I would rephrase it because it is not so obvious and also a bit scary. Maybe something like this?

> Retrieving a sample for an action creates new objects in the target system. If you want to avoid ....